modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -0,0 +1,141 @@
+{{>partial_header}}
+--package {{packageName}}
+
+{{#operations}}
+local http_request = require "http.request"
+local http_util = require "http.util"
+local dkjson = require "dkjson"
+local basexx = require "basexx"
+
+-- model import
+local {{{packageName}}}_{{classname}} = require "{{{packageName}}}.api.{{classname}}"
+
+local {{{packageName}}}= {}
+local {{{packageName}}}_mt = {
+  __name = "{{{classname}}}";
```

Comment:
Subjective question: indentation. 
Most lua is indented in tabs or 4 spaces; however some people use 3 spaces due to it being the default in emacs; and the rare person use 8 spaces. However, 2 spaces is pretty rare.
May I suggest swapping to tabs?